lang/cs/Org.Apache.REEF.Examples.HelloREEF/HelloREEF.cs

```diff
@@ -63,7 +72,13 @@ private void Run()
                 .SetJobIdentifier("HelloREEF")
                 .Build();
 
-            _reefClient.SubmitAndGetDriverUrl(helloJobSubmission);
+            var jobSubmissionResult = _reefClient.SubmitAndGetJobStatus(helloJobSubmission);
+
+            var appId = jobSubmissionResult.AppId;
+            var state = _reefClient.GetJobFinalStatus(appId);
+
+            Logger.Log(Level.Info, string.Format(CultureInfo.CurrentCulture,
+              "Job is submited with id: {0} and state: {1}", appId, state));
```

Comment:
Do we actually intend to change HelloREEF? Seems unnecessary bloat for HelloWorld example.